cc/position.h

```diff
@@ -30,6 +30,10 @@
 #include "cc/stone.h"
 
 namespace minigo {
+ namespace position {
+  inline_vector<Coord, 4>& GetNeighborCoords(Coord c);
```

Comment:
Return a const reference

Reply:
Done.

Reply:
I don't see the change, do you need to push or something?